packages/@okta/vuepress-site/docs/guides/migrate-to-idx/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Migrate to Okta Identity Engine (IDX)
+meta:
+  - name: description
+    content: Our guide shows how to migrate your organization and clients to IDX 
+layout: Guides
+---
+
+## Enable IDX for your organization
+
+# TODO - how to enable IDX?
```

Comment:
@wdawson what is there to say here?

Reply:
I think the other SDK teams have some boilerplate already so I would use the same. Something to the effect of "you must contact your account representative."

Reply:
added some boilerplate